### PR TITLE
Cassandra 19050 5.0

### PR DESCRIPTION
**Pull request by bereng**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

**Comment by bereng** [comment]
It is very tempting to try to extract the repeated code into a function. But given the variations on the format string, the sequential numbering, etc it becomes a mess so I left it as it is...

**Comment by bereng** [comment]
Given CQLTester is widely used and sometimes cross-referenced the [j11](https://app.circleci.com/pipelines/github/bereng/cassandra/1121/workflows/3df2bb12-9795-4d4f-889a-1776fdffe8ad) CI is an extensive one. The [j17](https://app.circleci.com/pipelines/github/bereng/cassandra/1121/workflows/6b44a166-eaa5-4a1d-830c-25d82460bb5f) is just junit.

File: test/unit/org/apache/cassandra/tools/JMXCompatabilityTest.java [hunk 1]
```diff
@@ -72,6 +72,7 @@ public class JMXCompatabilityTest extends CQLTester
     @BeforeClass
     public static void setup() throws Exception
     {
+        decorateCQLWithTestNames = false;
```
Comment (smiklosovic) [inline test/unit/org/apache/cassandra/tools/JMXCompatabilityTest.java:75]: @bereng could you rename this whole test while you are on it? It is `JMXCompatibilityTest` not `JMXCompatabilityTest`.

Comment (bereng) [reply test/unit/org/apache/cassandra/tools/JMXCompatabilityTest.java:75]: lol will do!

**Review by michaelsembwever — APPROVED** [review]

**Comment by bereng** [comment]
Ci [j11](https://app.circleci.com/pipelines/github/bereng/cassandra/1133/workflows/a1309702-40fa-478f-a033-a4b224449db3) [j17](https://app.circleci.com/pipelines/github/bereng/cassandra/1133/workflows/85236296-3588-4438-a8de-18e15ecba331)

Known flakies. LGTM.

**Pull request closed without merging**